extensions/wikia/SpitfiresContributionExperiments/Hooks.class.php

```diff
@@ -39,7 +39,11 @@ public function onUserLoadFromHeliosToken( \User $user ) {
 	}
 
 	public function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
-		if ( isset( $_COOKIE[ self::NEWLY_REGISTERED_USER ] ) || isset( $_COOKIE[ self::WITHOUT_EDIT_USER ] ) ) {
+		$user = \RequestContext::getMain()->getUser();
```

Comment:
`OutputPage` is a context source, so you can just use `$out->getUser();` (and actually `Skin` is too so `$skin->getUser();` would also work :)).